backend/src/main/java/com/example/skicurort/curort/CurortControler.java

```diff
@@ -0,0 +1,57 @@
+package com.example.skicurort.curort;
+
+import static com.example.skicurort.curort.CurortMapper.*;
+
+import com.example.skicurort.exception.NoIdException;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/curort")
+public class CurortControler {
+
+  private final CurortService curortService;
+
+  @GetMapping("/")
+  List<CurortDTO> getAll() {
+    return mapCurortsToCurortDTOs(curortService.getAll());
+  }
+
+  @PostMapping("/")
+  public ResponseEntity<CurortDTO> save(@RequestBody CurortDTO curortDto) {
+
+    curortService.save(mapCurortDTOToCurort(curortDto));
+    return ResponseEntity.ok(curortDto);
+  }
+
+  @GetMapping("/{id}")
+  CurortDTO findByid(@PathVariable Long id) throws NoIdException {
+    return mapCurortToCurortDTO(curortService.findById(id));
+  }
+
+  @PatchMapping("/{id}")
+  public ResponseEntity<String> update(@RequestBody CurortDTO curortDTO, @PathVariable Long id) {
+
+    curortService.updateDto(curortDTO, id);
+
+    return ResponseEntity.ok("Done " + curortDTO);
+  }
+
+  @DeleteMapping("/{id}")
+  public String delete(@PathVariable Long id) throws NoIdException {
+
+    curortService.delete(id);
+    return "Delete object with id: " + id + " sucesfull";
```

Comment:
metody controlera restowego powinny zwracać obiekty (json). te komentarze są raczej niepotrzebne i ciężko je będzie zmapować na froncie. zamiast tego mozemy bazować na kodach HTTP (np. 200 OK itp)

Reply:
Oczywiście słuszna uwaga troche z rozpędu tak mi wyszło. pozmieniam to tylko musze sobie powtórzyć temat

Reply:
zrobiłem